README.md

```diff
@@ -1,14 +1,58 @@
 # rivet-add-on-boilerplate
-A starting point for developing Rivet add-ons
+A boilerplate for Rivet add-ons. Replace the contents of this README with information about your add-on, following the template in README.template.md
 
 [Download](https://github.com/indiana-university/rivet-add-on-boilerplate/archive/master.zip) | [Demo](https://indiana-university.github.io/rivet-add-on-boilerplate/)
 
 ## Getting started
-1. Clone or download this repo
-2. Run `npm install`
-3. Run `npm run start` to start devlopment server and watch for changes
+This repo is meant to be cloned, and used as a starting point for a Rivet add-on repo. It contains the javascript and css for the add-on, as well as an example of the HTML required by the component. There are a few steps to get this repo ready to be used as a base.
+
+### 1. Clone this repo
+Clone this repo `https://github.iu.edu/UITS/rivet-bopilerplate` to the machine where you'll be developing your add-on.
+
+Do a find-and-replace of the word "add-on-boilerplate" with the name of your add-on. A find-and-replace should be replaced on *file contents*, as well as for the filenames below
+
+```
+src/js/rivet-add-on-boilerplate.js
+src/sass/rivet-add-on-boilerplate.scss
+src/sass/components/_rivet-add-on-boilerplate.scss
+```
+
+This can be done with the following commands on a unix based system
+
+```
+# TODO write this
+```
+
+### 2. Include the CCS and JavaScript in your page
+After renaming the files, edit `src/index.html` to use the updated names
+
+```html
+<link rel="stylesheet" href="dist/css/rivet-add-on-boilerplate.min.css">
+<script src="dist/js/rivet-add-on-boilerplate.min.js"></script>
+```
+
+### 3. Add the markup to your HTML
+Add your markup to `src/index.html`, replacing the HTML comment `<!-- PLACE BOILERPLATE EXAMPLE HERE -->`. Include as many examples as are helpful to convey the variations and options of the component.
+
+This is also a great place to add documentation, accessibility notes, or microcopy guidelines, as the `index.html` file will be built to the `docs` folder and can be published with GitHub Pages.
+
+
+### 4. Write the documentation
```

Comment:
"the documentation" is a little ambiguous, what documentation specifically? 

Maybe this header would read "Write add-on documentation, accessibility notes, and microcopy guidelines"